packages/router/src/router-context.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useReducer, createContext, useContext } from 'react'
+
+import type { AuthContextInterface, useAuth } from '@redwoodjs/auth'
+
+import { ParamType } from './internal'
+
+const DEFAULT_PAGE_LOADING_DELAY = 1000 // milliseconds
```

Comment:
Underscores can be used as numeric separators, and make them easier to read
```suggestion
const DEFAULT_PAGE_LOADING_DELAY = 1_000 // milliseconds
```

Reply:
👍

Reply:
Actually...

I know this should work. But 🙁 

![image](https://user-images.githubusercontent.com/30793/111609072-34991500-87da-11eb-89fa-41c502564431.png)

Reply:
Lame, I'm assuming some sort of babel setting.